api/src/pcapi/routes/internal/testing.py

```diff
@@ -13,10 +13,10 @@
 logger = logging.getLogger(__name__)
 
 
-@apis.public_api.route("/testing/features", methods=["PATCH"])
-@spectree_serialize(on_success_status=204, api=apis.api)
+@public_api.route("/testing/features", methods=["PATCH"])
+@spectree_serialize(on_success_status=204)
 def set_features(body: serializers.FeaturesToggleRequest) -> None:
-    if not (settings.IS_DEV or settings.IS_RUNNING_TESTS):
+    if not (settings.ENABLE_TEST_ROUTES):
```

Comment:
[détail / style] Les parenthèses sont inutiles :

```suggestion
    if not settings.ENABLE_TEST_ROUTES:
```